api/passthrough/getImageByAmiId.js

```diff
@@ -107,11 +107,7 @@ function _getImageFromAmiId(bag, next) {
   bag.awsAdapter.EC2.getImages(bag.amiId,
     function (err, images) {
       if (err)
-        return next(
```

Comment:
With this horn error was coming twice one form api and same from UI, so it was required to remove from any one place.
If we want Acterr we can remove the horn from the UI, should I remove from UI instead of this ?